### PR TITLE
sysdeps: Add Keyronex sysdeps

### PR DESCRIPTION
**Pull request by netbsduser**
This adds the initial set of sysdeps for Keyronex.

### CODE REVIEW

**Review by no92 — APPROVED** [review]
LGTM

**Comment by mintsuki** [comment]
LGTM

File: sysdeps/keyronex/generic/entry.cpp [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	}
+*/
+
+	switch ((uintptr_t)sa) {
+		// DFL
+		case (uintptr_t)(-2):
+			mlibc::infoLogger() << "mlibc: Unhandled signal " << which << frg::endlog;
+			mlibc::sys_exit(128 + which);
+		// IGN
+		case (uintptr_t)(-3):
```
Comment (Dennisbonke) [inline sysdeps/keyronex/generic/entry.cpp:108]: Why not use the defines here instead of magic numbers?

**Review by Dennisbonke — COMMENTED** [review]
LGTM, but one comment

**Pull request merged**